src/app/context/AccountContext.tsx

```diff
@@ -121,11 +121,14 @@ export function AccountProvider({ children }: { children: React.ReactNode }) {
         }
       })
       .catch((e) => {
-        toast.error(`An unexpected error occurred (${e.message})`);
+        toast.error(
+          `AccountContext: An unexpected error occurred (${e.message})`
```

Comment:
why do we write "AccountContext" there? this is not really helpful for the user I guess.

Reply:
So far this was helpful for us. Doubt the user cares or has a worse experience because of these tbh

Reply:
why is it helpful for us? why not use `console.error(e)`?